### ISSUE TITLE
Eligibility Form_DPGA website mobile compatibility

### ISSUE THREAD

**Issue by PelinSmines** [DPGAlliance/publicgoods-scripts#232]
[Eligibility page ](https://digitalpublicgoods.net/eligibility/) has some UI issues:
The form frame isn't centred on the page. It's aligned on the right. See pic: 
![IMG_2304](https://github.com/DPGAlliance/publicgoods-scripts/assets/123653036/7658e091-bee7-469a-8f6b-34de7b2e520d)

When click on Get started, the next page; the first question of the form opens by showing the middle of the page, not the start of the question.
the Process bar (Question 5 of 7) shows 7 questions to begin with but then after 7th question it turns into 10 question-form. It shows: Question 8 of 10.
Question 5, 6, 8 in the eligibility form has text that is overlapping with Yes answer option. See pic: 
![IMG_2303](https://github.com/DPGAlliance/publicgoods-scripts/assets/123653036/cd5c74a4-3428-495a-ad95-b2d01dd9e068)

**Comment by law909** [comment]
> the Process bar (Question 5 of 7) shows 7 questions to begin with but then after 7th question it turns into 10 question-form. It shows: Question 8 of 10.
The seventh question is "Are you the developer or owner of this digital solution?". if you answer yes then you have to answer the next three questions. If you answer no, then the next three questions don't apply to you, so you don't have to answer those.
I think this is a feature not a bug @PelinSmines

**Comment by law909** [comment]
Other issues fixed

**Comment by PelinSmines** [comment]
You are right @law909 :) I tried different combos before to come to different end page and didn't pay attention that 7th question is creating that condition to proceed. Thanks.
